### PR TITLE
Clearer CTAs on Homepage

### LINKED ISSUE #199: Clearer CTAs on Homepage
### Story

_No response_

### Description

Currently, we have the following CTAs:

Screenshot 2022-10-16 at 17 44 56

at the point of first arrival, new customers/visitors are unlikely to have knowledge of our different offerings, as such "Sign Up" and "Install" are vague and confusing - do I need to sign up in order to install? 

We should have clearer distinction between these actions, as such I propose the following:

Screenshot 2022-10-16 at 17 46 11

Where the "FlowForge Cloud" is, I know, going to open a can of worms in itself, but let's restrain this issue to the clearer messaging component.

_note: the design changes here also include/assume merging of https://github.com/flowforge/website/pull/200_

### PR DESCRIPTION
**Pull request by joepavitt**
Closes #199 

**Before:**

Screenshot 2022-10-16 at 17 44 56

**After:**

Screenshot 2022-10-16 at 17 46 11

### CODE REVIEW

**Comment by joepavitt** [comment]
@ZJvandeWeg what is an appropriate alternative to "Join FlowForge Cloud" we can use?

**Comment by knolleary** [comment]
Just saw this PR *after* commenting on the related issue.

I suggest being consistent with the CTA text on the product page:

image

**Comment by joepavitt** [comment]
They are still too wordy and technical. I also want to change the CTAs on the Product Page. Don't think it's clear from a product/branding perspective either - although I will settle on them as a bridge to something better.

**Comment by joepavitt** [comment]
Updated to align Product & Homepage CTA:

Screenshot 2022-10-17 at 11 34 00

**Review by knolleary — APPROVED** [review]
